LICENSES/MIT.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) <year> <copyright holders>
```

Comment:
Copyright © 2018 - 2019 SamBrishes, pytesNET <info@pytes.net>
would be the correct data